src/components/AppHeader.vue

```diff
@@ -77,10 +77,11 @@ import VHeader from "./lib/VHeader/VHeader.vue";
 import { useRoute, useRouter } from "vue-router";
 import { useStore } from "vuex";
 import CancelIcon from "@/assets/iconography/cancel.svg";
+import CloseIcon from "@/components/icons/CloseIcon.vue";
 
 export default {
   name: "AppHeader",
-  components: { VHeader },
+  components: {CloseIcon, VHeader },
```

Comment:
Add consistent spacing between the braces:

```suggestion
  components: { CloseIcon, VHeader },
```